### DISCUSSION TITLE
Crashing when loading a character

### DISCUSSION THREAD

**Discussion by TrashSamaUwU** [ikemen-engine/Ikemen-GO#2086] [Q&A] [answered]
it crashes when selecting a character i know that its a syntax error but i cant find the problem and even if i did i dont entirely know how to fix it
Log:

.\external\script\start.lua:1403: chars\Wolverine (Symbiote)/Hypers.cns:3159:
6, Hit
priority: Invalid data: ,
stack traceback:
	[G]: in function 'game'
	.\external\script\start.lua:1403: in function 'f_game'
	.\external\script\start.lua:1839: in function 'launchFight'
	external/script/default.lua:2: in function <external/script/default.lua:0>
	.\external\script\start.lua:1457: in function 'f'
	external/script/main.lua:3069: in function 'loop'
	external/script/main.lua:3189: in function 'loop'
	external/script/main.lua:4156: in main chunk
	[G]: ?

**Comment by lokpro** [comment]
the error is on line 3159 of  chars\Wolverine (Symbiote)/Hypers.cns

first find the line using text editor that shows line numbers.

then check the syntax

**Reply by TrashSamaUwU** [reply]
there nothing even written on that line could that be the issue

**Reply by potsmugen** [reply]
Look around that line. It's not an exact number. There's probably a PlaySnd and HitDef mixed together.

**Comment by lokpro** [comment]
where to download this char?

**Comment by TrashSamaUwU** [comment]
here's the link i did change the name of the folder and def for organization

https://mugenarchive.com/forums/downloads.php?do=file&id=1676-wolvenom-darktalbain-cray

**Comment by lokpro** [comment, marked as answer]
found 7 like this. need to uncomment all 
;[State 3004, 1] ... 
;...
;getpower ...
to get it to work.


```ini
[State 0, PlaySnd]
type = PlaySnd
trigger1 = time = 0
value = 100,5


;[State 3004, 1]
;type = HitDef
;trigger1 = Time = 0
;attr = A, HA			;Attribute: Standing, Normal Attack
;damage = 20, 8			;Damage that move inflicts, guard damage
;getpower = 0
animtype = Up			;Animation type: Light, Medium, Heavy, Back (def: Light)
guardflag = MA			;Flags on how move is to be guarded against
hitflag = MAF			;Flags of conditions that move can hit
priority = 6, Hit			;Attack priority: 0 (least) to 7 (most), 4 default
pausetime = 0, 5			;Time attacker pauses, time opponent shakes
guard.pause = 13
sparkno = 3;s985				;Spark anim no (Def: set above)
sparkxy = -20, -44		;X-offset for the "hit spark" rel. to p2,
hitsound = s300, 20			;Sound to play on hit
guardsound = s6300, 0			;Sound to play on guard
ground.type = high		;Type: High, Low, Trip (def: Normal)
ground.slidetime = 10		;Time that the opponent slides back
ground.hittime  = 30		;Time opponent is in hit state
ground.velocity = -0.2, -6		;Velocity at which opponent is pushed
;airguard.velocity = -3,-3	;Guard velocity in air (def: (air.xvel*1.5, air.yvel/2))
air.type = high			;Type: High, Low, Trip (def: same as ground.type)
air.velocity = -0.2, -6		;X-velocity at which opponent is pushed,
air.hittime = 30			;Time before opponent regains control in air
air.fall = 1
fall = 1
fall.recover = 0
envshake.time = 5
envshake.ampl = 15
```
